src/components/POLine/Item/ItemForm.js

```diff
@@ -31,25 +41,50 @@ class ItemForm extends Component {
     stripes: PropTypes.object.isRequired,
     dispatch: PropTypes.func.isRequired,
     change: PropTypes.func.isRequired,
+    identifierTypes: PropTypes.arrayOf(PropTypes.object),
+    initialValues: PropTypes.object,
   };
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      instanceId: '',
+      title: '',
+      publisher: '',
+      publicationDate: '',
+      edition: '',
+      contributors: [],
+      productIds: [],
+    };
+  }
+
   onAddInstance = (instance) => {
-    const { dispatch, change } = this.props;
-    const { contributors, editions, publication, title } = instance;
+    const { dispatch, change, identifierTypes } = this.props;
+    const { contributors, editions, publication, title, identifiers, id } = instance;
+    const inventoryData = { instanceId: id };
```

Comment:
you override `id` here, is it by intention?

Reply:
It is always a new `id` that user get from inventory